retrieval/targetmanager.go

```diff
@@ -143,6 +143,23 @@ func (tm *TargetManager) reload() {
 	}
 }
 
+// TargetMap returns map of active and dropped targets and their corresponding scrape config job name.
+func (tm *TargetManager) TargetMap() map[string][]*Target {
+	tm.mtx.RLock()
+	defer tm.mtx.RUnlock()
+
+	targetsMap := make(map[string][]*Target)
+	for jobName, ps := range tm.targetSets {
+		ps.sp.mtx.RLock()
+		targetsMap[jobName] = append(targetsMap[jobName], ps.sp.droppedTargets...)
```

Comment:
It'd probably be better UI wise for the dropped targets to go at the end.

Reply:
Good point. I've amended this now.